kit/src/main/java/com/oracle/javafx/scenebuilder/kit/editor/panel/library/manager/LibraryDialogController.java

```diff
@@ -91,12 +93,13 @@ public class LibraryDialogController extends AbstractFxmlWindowController {
     
     private ObservableList<DialogListItem> listItems;
 
-    private Runnable onAddJar;
-    private Runnable onAddFolder;
+    private ReturningRunnable<Boolean> onAddJar;
```

Comment:
Please consider replacing the `ReturningRunnable` with JDKs default `java.util.function.Supplier` interface.